assets/js/modules/tagmanager/components/common/WebContainerSelect.test.js

```diff
@@ -284,48 +284,36 @@ describe( 'WebContainerSelect', () => {
 		).toHaveTextContent( /Container/i );
 	} );
 
-	it( 'should be labeled as "Web Container" in a secondary AMP context', () => {
-		const { account, containers } = factories.buildAccountWithContainers();
-		const accountID = account.accountId; // eslint-disable-line sitekit/acronym-case
-		registry.dispatch( MODULES_TAGMANAGER ).setAccountID( accountID );
-		registry
-			.dispatch( MODULES_TAGMANAGER )
-			.receiveGetAccounts( [ account ] );
-		registry
-			.dispatch( MODULES_TAGMANAGER )
-			.finishResolution( 'getAccounts', [] );
-		registry
-			.dispatch( MODULES_TAGMANAGER )
-			.receiveGetContainers( containers, { accountID } );
-		registry
-			.dispatch( MODULES_TAGMANAGER )
-			.finishResolution( 'getContainers', [ accountID ] );
-		provideSiteInfo( registry, { ampMode: AMP_MODE_SECONDARY } );
-
-		const { container } = render( <WebContainerSelect />, { registry } );
-
-		expect(
-			container.querySelector( '.mdc-floating-label' )
-		).toHaveTextContent( /Web Container/i );
-	} );
-
-	it( 'should render nothing in a primary AMP context', () => {
-		const account = factories.accountBuilder();
-		registry
-			.dispatch( MODULES_TAGMANAGER )
-			.receiveGetAccounts( [ account ] );
-		provideSiteInfo( registry, { ampMode: AMP_MODE_PRIMARY } );
-
-		const { queryByRole, container } = render( <WebContainerSelect />, {
-			registry,
-		} );
+	it.each( [ AMP_MODE_PRIMARY, AMP_MODE_SECONDARY ] )(
+		'should be labeled as "Web Container" in a %s AMP context',
+		( mode ) => {
```

Comment:
It's a small change, but let's rename `mode` to `ampMode`, it's more explicit and means the `provideSiteInfo()` args below can use the shorthand `{ ampMode }` :)

```suggestion
		( ampMode ) => {
```